models/intermediate/hubspot/int_rag_hubspot__deal_comment_document.sql

```diff
@@ -177,4 +178,5 @@ truncated_comments as (
 )
 
 select *
-from truncated_comments
\ No newline at end of file
+from truncated_comments
+where comment_markdown is not null
```

Comment:
I found that there were nearly 300k records in this output model that had no RAG relevant data associated with them. I'm not sure if this is how customer environments will look, but it was resulting in some pretty skewed data when testing on our test schemas internally. I don't see this being an issue as we shouldn't include any deal information that isn't useful for a RAG use case.

Reply:
Sounds good!